src/addrparse.rs

```diff
@@ -1062,5 +1062,26 @@ mod tests {
                 .unwrap()
             )])
         );
+
+        let (parsed, _) = crate::parse_header(
+            b"To: foo <foo@example.org>,?UTF-8?B?Zm9v8J+Qm2Jhcg==?= <bar@example.org>",
```

Comment:
Is it intentional that there's no `=` between the `,` and the `?UTF` bits? Because that's a pretty bad spec violation and I wouldn't want to try and detect that. Even if I add the `=`, mailparse doesn't handle this correctly, so there's still a bug that I need to fix, but I wanted to confirm that your original thing has the `=` character and the omission here is a typo.